server/db.go

```diff
@@ -360,57 +546,95 @@ func (c *Server) rotateIssuers() (err error) {
 	fetchedIssuers := []issuer{}
 	err = tx.Select(
 		&fetchedIssuers,
-		`SELECT * FROM issuers 
+		`SELECT * FROM v3_issuers
 			WHERE expires_at IS NOT NULL
 			AND rotated_at IS NULL
```

Comment:
The column rotated_at dosent exist on v3_issuers should it be last_rotated_at

Reply:
https://github.com/brave-intl/challenge-bypass-server/pull/145/commits/2095b05667b6175637a40aead6ea3dbde57e4a01